server/server_test.go

```diff
@@ -309,44 +309,51 @@ func TestGinPostCreateInteraction(t *testing.T) {
 	}
 }
 func TestGinGetGame(t *testing.T) {
+	createTestGame := func(t *testing.T, cs *cribbageServer, pIDs []model.PlayerID) model.Game {
```

Comment:
🌵 📖 now that this is an anonymous func, it has access to `t *testing.T` from the `func TestGinGetGame`, so it doesn't need to accept `t` as an input param. 🤷